### PR TITLE
remove depdendency on Ubahn-api

### PR DESCRIPTION
**Pull request by Schpotsky**

### CODE REVIEW

File: src/models/JobCandidate.js [hunk 1]
```diff
@@ -49,7 +49,12 @@ module.exports = (sequelize) => {
       userId: {
         field: 'user_id',
         type: Sequelize.UUID,
-        allowNull: false
+        allowNull: true // temporarly mark it as nullable until complete removal
```
Comment (testflyjets) [inline src/models/JobCandidate.js:52]: Please add a subtask to the Jira story for this work to remove `userId` and change the nullable for `tc_user_id`

**Review by testflyjets — APPROVED** [review]
I love seeing code removed!

File: src/models/JobCandidate.js (hunk 1 above)
Comment (Schpotsky) [reply src/models/JobCandidate.js:52]: Done https://topcoder.atlassian.net/browse/TSJR-325
It is added into Sprint 8

**Comment by kkartunov** [comment]
Closing this one as work will be handled in https://github.com/topcoder-platform/taas-apis/pull/710

**Pull request closed without merging**